core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -452,8 +452,8 @@ object KafkaConfig {
 
   /* Documentation */
   /** ********* Zookeeper Configuration ***********/
-  val ZkConnectDoc = "Zookeeper host string"
-  val ZkSessionTimeoutMsDoc = "Zookeeper session timeout"
+  val ZkConnectDoc = "String of ZooKeeper host and port values in the format zkhost:port[,zkhost:port*/znode]. If znode is not provided ZooKeeper root znode will be used by default"
```

Comment:
maybe we reuse the description given in the "zookeeper.connect"  property of old consumer configs docs here.
http://kafka.apache.org/documentation/#oldconsumerconfigs

Reply:
Thanks @omkreddy for the comment. Since ``consumer`` api is moving away from using ``ZooKeeper`` do we need to include that in the description as was done in the old doc. If not, the changes in the PR seem to reflect what was in the old doc. Let me know.

Reply:
@bijugs  yeah, I was suggesting to use similar description as old doc. or you can use the word "chroot" in-place of "zknode".  suffix in the zk connection string is generally referred as "chroot".

Reply:
@omkreddy , made changes to the description based on your suggestions. Let me know whether it looks fine. Thanks.